src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/ModelBindingHelper.cs

```diff
@@ -553,7 +554,7 @@ public static class ModelBindingHelper
         }
 
         // Internal for tests
-        internal static string GetPropertyName(Expression expression)
+        public static string GetPropertyName(Expression expression)
```

Comment:
Because it was used by unit tests - same as before, need to follow up on confirming policy